test/experimental/features_config/install.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package features_config
+
+import (
+	"context"
+
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/manifest"
+)
+
+const ConfigMapName = "config-features"
+
+func Install(ctx context.Context, t feature.T) {
```

Comment:
Can you add a test for this like in here? : https://github.com/knative-sandbox/eventing-kafka/blob/b42f8dff9753f65aa41a3459a75f898d8366578c/test/rekt/resources/kafkachannel/kafkachannel_test.go

Reply:
Oh, no need. There are nothing configurable in the YAML anyway. Never mind.